app/views/layouts/nav/staff/_program_subnav.html.haml

```diff
@@ -0,0 +1,19 @@
+.navbar.navbar-default.program-subnav
+  .container-fluid
+    %ul.nav.navbar-nav
```

Comment:
selection is first because that's your working program dashboard and your primary landing page until the program selection is complete.  Proposals is a secondary link that you use to find more to add into the working program.